packages/pyroscope-flamegraph/src/Toolbar.tsx

```diff
@@ -33,8 +42,8 @@ export const TOOLBAR_MODE_WIDTH_THRESHOLD = 900;
 
 export type ShowModeType = ReturnType<typeof useSizeMode>;
 
-export const useSizeMode = (target: React.RefObject<HTMLDivElement>) => {
-  const [size, setSize] = React.useState<'large' | 'small'>('large');
+export const useSizeMode = (target: RefObject<HTMLDivElement>) => {
```

Comment:
also guys what do you think about this: if we have this "more" button functionality we can go towards removing `size` logic, where we have 

large
<img width="847" alt="Screenshot 2022-11-23 at 11 11 17" src="https://user-images.githubusercontent.com/47758224/203520809-8097cf92-6ce3-40be-a889-17b0b457b064.png">

and small
<img width="819" alt="Screenshot 2022-11-23 at 11 11 40" src="https://user-images.githubusercontent.com/47758224/203520897-1829d74a-8bb1-48e8-85c6-cfd2dd6a1486.png">

toolbars depending on width and just show **large** option with additional "more" button (3dot) 

cc @eh-am @Rperry2174

Reply:
I agree, that simplifies a lot!